src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -531,9 +530,9 @@ void ov::npuw::LLMInferRequest::infer_prefill(ov::SoPtr<ov::ITensor> input_ids,
     if (m_lm_head_request) {
         LOG_DEBUG("Calling inference for LM head model.");
         m_lm_head_request->infer();
-        m_logits = m_lm_head_request->get_tensor(m_lm_head_logits_port);
+        update_out_tensors_from(m_lm_head_request);
```

Comment:
Wrong. Only `logits` will be in LM head, other outputs should be gathered from prefill.